include/miopen/miopen.h

```diff
@@ -583,6 +583,50 @@ typedef enum
  */
 MIOPEN_EXPORT miopenStatus_t miopenCreateTensorDescriptor(miopenTensorDescriptor_t* tensorDesc);
 
+/*! @brief Set the quantization scale of Tensor Descriptor
+ *
+ * API for setting the quantization scale of tensor descriptor.
+ * @param tensorDesc  Pointer to a tensor descriptor type
+ * @param quantScales Quantization scales
+ * @param len         Length of quantization scales buffer
+ * @return            miopenStatus_t
+ */
+MIOPEN_EXPORT miopenStatus_t miopenSetQuantizationScales(miopenTensorDescriptor_t tensorDesc,
+                                                         double* quantScales,
+                                                         size_t len);
+
+/*! @brief Set the quantization bias of Tensor Descriptor
+ *
+ * API for setting the quantization bias of tensor descriptor.
+ * @param tensorDesc  Pointer to a tensor descriptor type
+ * @param quantBiases Quantization biases
+ * @param len         Length of quantization biases buffer
+ * @return            miopenStatus_t
+ */
+MIOPEN_EXPORT miopenStatus_t miopenSetQuantizationBiases(miopenTensorDescriptor_t tensorDesc,
+                                                         double* quantBiases,
+                                                         size_t len);
+
+/*! @brief Get the quantization scale of tensor descriptor
+ *
+ * Interface for querying tensor quantization scale.
+ * @param tensorDesc   Tensor descriptor type (input)
+ * @param quantScales  Tensor quantization scales (output)
+ * @return             miopenStatus_t
+ */
+MIOPEN_EXPORT miopenStatus_t miopenGetQuantizationScales(miopenTensorDescriptor_t tensorDesc,
+                                                         double* quantScales);
```

Comment:
How does the user know how much memory to allocate for `quantScales` ?

Reply:
> How does the user know how much memory to allocate for `quantScales` ?

I will add another parameter that controls the memory size that users need to allocate.